documentation/enforce/README.md

```diff
@@ -12,8 +12,8 @@ You can consume the `enforce` function in two ways:
 ```js
 import passable, {enforce} from 'passable';
 
-Passable('enforcement', (pass) => {
-    pass('test', 'enforce example', () => {
+Passable('enforcement', (test) => {
```

Comment:
`Passable` -> `passable` right?

Reply:
This is a good comment, but not related to this specific PR. The docs issue needs to be addressed separately before pushing the next version.
👍 
Adding an issue.